catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -123,6 +123,7 @@
              description="Monitors directories to process content files.">
         <feature>apache-commons</feature>
         <bundle>mvn:ddf.catalog.core/catalog-core-camelcontext/${project.version}</bundle>
+        <bundle>mvn:com.google.code.gson/gson/${gson.version}</bundle>
```

Comment:
❓ This isn't being picked up from the CDM specific Itests. Anyone have ideas why?

Reply:
ContentDirectoryMonitorIT? That test does not start the feature, it starts the necessary bundles individually. 
https://github.com/codice/ddf/blob/master/catalog/core/catalog-core-directorymonitor/src/test/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitorIT.java#L308-L344

Reply:
Thanks that was it 😄